kentik_api_library/kentik_api/requests_payload/populators_payload.py

```diff
@@ -2,12 +2,64 @@
 from typing import Any, Dict, List, Optional
 
 from kentik_api.public.custom_dimension import Populator
+from kentik_api.public.errors import IncompleteObjectError
 from kentik_api.public.types import ID
 from kentik_api.requests_payload.conversions import convert, dict_from_json, from_dict
 
 # pylint: disable=too-many-instance-attributes
 
 
+@dataclass
+class PopulatorPayload:
```

Comment:
We already have top level populator payload dataclass  called `_Populator`. After this change we'll have 2, which is confusing. I don't think it is a right direction. We need at least some naming convention let reader understand purpose of those 2 payload classes.

Reply:
After the changes, PopulatorPayload and PopulatorGetPayload is acceptable for me.